src/commands/reset.php

```diff
@@ -38,6 +38,13 @@
 
 quietly_tear_down_stack();
 
+$lastruntime = root( '/.lastruntime' );
+echo "Removing {$lastruntime} ... ";
+echo ( ! file_exists( $lastruntime ) || unlink( $lastruntime ) ) ?
```

Comment:
This could be rewritten for readability:
`if( file_exists( $lastruntime ) && !unlink ( $lastruntime ) ) { ... }`
"If the file exists and cannot be removed" reads better, in my opinion.